pkg/testing/integration/program.go

```diff
@@ -68,6 +68,8 @@ const (
 
 const windowsOS = "windows"
 
+var ErrTestCanceled = errors.New("test canceled")
```

Comment:
I'm bikeshedding here, but maybe `ErrTestAlreadyFailed` would be a better name?  canceled sounds a bit like something a user did to me.

Reply:
I considered something similar (`ErrTestFailed`) but I convinced myself that the purpose of the error is to interrupt the flow of the `ProgramTest` func without logging the error.  Still I think I'll go back to `ErrTestFailed`.